app/assess/routes.py

```diff
@@ -557,7 +557,7 @@ def application(application_id):
         accounts_list=accounts_list,
         teams_flag_stats=teams_flag_stats,
         flags_list=flags_list,
-        current_user_role=g.user.highest_role,
+        current_user_role=g.user.highest_role_map,
```

Comment:
This isn't used anymore, think it's better to remove it?

Reply:
removed